tests/unit/core/data/test_dataset.py

```diff
@@ -24,6 +25,19 @@ def test_get_item(
         assert isinstance(item, data_entity_cls)
         fxt_mock_dm_subset.get.assert_called_once()
 
+        mocker.patch.object(dataset, "_get_item_impl", return_value=None)
+        with pytest.raises(RuntimeError):
```

Comment:
Is this about testing `max_refetch`? If so, please set it to small value such as `max_refetch=3` to keep the unit test computationally lighter as possible.

Reply:
Done by [0f6bcf7](https://github.com/openvinotoolkit/training_extensions/pull/2711/commits/0f6bcf7778a3299d61e3ce1a30b5550ecb719051)